src/components/HomePage.jsx

```diff
@@ -223,256 +297,253 @@ class HomePage extends Component {
+									<Button
+										variant="contained"
+										color="primary"
+										alignSelf="center"
+										className={classes.button}
+										startIcon={<HowToRegIcon />}
+										href={config.signUpURL}
+									>
+										Sign up for IN-CORE
+									</Button>
+								</div>
+								<Divider />
+							</div>
+							<div>
+								<h2>Learn IN-CORE</h2>
+								<p>
+									Gain a comprehensive understanding of IN-CORE by reviewing the&nbsp;
+									<Link href={config.incoreDocUrl} target="_blank" className={classes.link}>
+										step-by-step manual
+									</Link>
+									. To delve deeper into advanced IN-CORE topics, check out our&nbsp;
+									<Link href={config.incoreTutorialUrl} target="_blank" className={classes.link}>
+										detailed tutorials
+									</Link>
+									.
+								</p>
+								<div className={classes.buttonDiv}>
+									<Button
+										variant="contained"
+										color="primary"
+										alignSelf="center"
+										href={config.incoreDocUrl}
+										className={classes.button}
+										startIcon={<BookIcon />}
+									>
+										IN-CORE Manual
+									</Button>
+									<Button
+										variant="contained"
+										color="primary"
+										alignSelf="center"
+										href={config.incoreTutorialUrl}
+										className={classes.button}
+										startIcon={<SchoolIcon />}
+									>
+										IN-CORE Tutorial
+									</Button>
+								</div>
+								<Divider />
+							</div>
+							<div>
+								<h2>Get Help</h2>
+								<p>
+									Got questions or need assistance? There are three ways to reach out for help:&nbsp;
+									<Link href={config.incoreFAQUrl} target="_blank" className={classes.link}>
+										review our FAQs for the most common questions
+									</Link>
+									, &nbsp;
+									<Link href={config.slackInvitationLink} target="_blank" className={classes.link}>
+										join the IN-CORE Slack channel
+									</Link>
+									, or&nbsp;
+									<Link href={`mailto:${config.mailingList}`} className={classes.link}>
+										email our dev team
+									</Link>
+									.
+								</p>
+								<div className={classes.buttonDiv}>
+									<Button
+										href={config.incoreFAQUrl}
+										variant="contained"
+										color="primary"
+										className={classes.button}
+										startIcon={<QuestionAnswerIcon />}
+									>
+										Read FAQs
+									</Button>
+									<Button
+										href={config.slackInvitationLink}
 										target="_blank"
-										variant="contained" color="primary"
+										variant="contained"
+										color="primary"
 										className={classes.button}
-										style={{marginRight: "1em"}}
-								>
-									<img src="/public/slack-logo.png" style={{ height: "1em", marginRight: "8px"}}/>
-									Join Slack
-								</Button>
-								<Button href={`mailto:${config.mailingList}`}
+										style={{ marginRight: "1em" }}
```

Comment:
You might want to remove the marginRight to be consistent with the space in between 2 buttons
<img width="295" alt="image" src="https://github.com/IN-CORE/incore-ui/assets/13950475/139c62b7-7ad8-4704-91bf-1ad40cef21ef">